pkg/kubelet/kubelet.go

```diff
@@ -50,7 +50,7 @@ import (
 	"k8s.io/client-go/util/clock"
 	"k8s.io/client-go/util/flowcontrol"
 	"k8s.io/client-go/util/integer"
-	"k8s.io/kubernetes/cmd/kubelet/app/options"
+	"k8s.io/kubernetes/cmd/kubelet/app/options" // I'm sorry :(. Options considered: Add this (sort-of-cyclic) dep, add more args to NewMainKubelet, define flags for cmd somewhere outside cmd... I am open to suggestions.
```

Comment:
After thinking about this, I think I'm going to add more args to NewMainKubelet instead. I really don't like this cyclic relationship.